tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm

```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright © 2016 SUSE LLC
```

Comment:
permission change is planned ? what the reason for this ? 
(just merge @okurz  PR where he fixes set of files to actually have 10644)

Reply:
I was planning to change the copyright and make it consistent with other files.  Since the copyright in most other x11regression files is 2016 - 2017. 
Sorry I forgot it's already 2018 not 2017.

Reply:
Fix the  permission back to 10644

Reply:
fixed (again) in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4155